solutions/9.js

```diff
@@ -0,0 +1,10 @@
+const greatCommDenom = (num1, num2, gcd, i=0)=>{  
+  if(i > num1 && i > num2){
```

Comment:
You should have a variable has a value of the smaller number between `num1` and `num2`.